prow/gerrit/adapter/adapter.go

```diff
@@ -328,20 +328,49 @@ func (c *Controller) processChange(logger logrus.FieldLogger, instance string, c
 
 	// comment back to gerrit if Report is set for any of the jobs
 	var reportingJobs int
-	var message string
+	var jobList string
 	for _, job := range triggeredJobs {
 		if job.report {
-			message += fmt.Sprintf("\n  * Name: %s", job.name)
+			jobList += fmt.Sprintf("\n  * Name: %s", job.name)
 			reportingJobs++
 		}
 	}
 
 	if reportingJobs > 0 {
-		message = fmt.Sprintf("Triggered %d prow jobs (%d suppressed reporting):", len(triggeredJobs), len(triggeredJobs)-reportingJobs) + message
+		message := fmt.Sprintf("Triggered %d prow jobs (%d suppressed reporting): ", len(triggeredJobs), len(triggeredJobs)-reportingJobs)
+		// If we have a Deck URL, link to all results for the CL, otherwise list the triggered jobs.
+		link, err := deckLinkForPR(c.config().Gerrit.DeckURL, refs)
+		if err != nil {
+			logger.WithError(err).Error("Failed to generate link to job results on Deck.")
+		}
+		if link != "" && err == nil {
+			message = message + link
```

Comment:
Sorry missed this yesterday: should message include jobList also in this case?

Reply:
I intentionally omitted the list of job names in favor of the link. We print the count of triggered jobs in either case, but I figured if we have the link we can reduce the size of the bot comments to make things more readable. I don't think users typically care which presubmits are triggered, just that they are triggered. Following the link will lead to specifics if they are needed.
WDYT?

Reply:
Makes sense. It probably doesn't make that much sense printing out each job if they are not clickable, so lgtm

Reply:
This also initially seemed odd to me but I think that's just my change aversion bias. Making these messages shorter SGTM.